docs/README.md

```diff
@@ -19,6 +19,7 @@ Read license for more details.
   - Volumes
   - Public Ips
   - Snapshots (🚨 Warning: snapshot computation is currently known to be over-priced.)
+  - Dedicated Vm (🚨 Warning: dedicated vm computation can be over-priced depend on the situation (Look at example of Pricing for Dedicated VMs [Outscale API](https://docs.outscale.com/api) .)
```

Comment:
"overpriced"  and "depending"

Reply:
The suitable link is this one, no ? https://docs.outscale.com/en/userguide/Getting-the-Price-of-Your-Resources.html#_calculating_the_price_of_dedicated_vms

Reply:
Done